pkg/util/cgroup/cgroups.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+//
+// ============================================================
+// Forked from https://github.com/KimMachineGun/automemlimit
```

Comment:
`github.com/KimMachineGun/automemlimit` requires go >= 1.19, so fork the cgroup part to fit go 1.18 in release-6.1